src/data/project-main-content/newrelic-newrelic-ruby-agent.mdx

```diff
@@ -0,0 +1,15 @@
+---
+path: "/projects/newrelic/newrelic-ruby-agent"
+date: "07/13/2020"
+title: "New Relic Ruby Agent"
```

Comment:
Modify the `path` slightly to match new name of the project json file. Also, there's a `projectConfig` setting that explicitly maps the project's json file.

```suggestion
path: "/projects/newrelic/newrelic-newrelic-ruby-agent"
date: "07/13/2020"
title: "New Relic Ruby Agent"
projectConfig: "src/data/projects/newrelic-newrelic-ruby-agent.json"
```